### PR TITLE
docs(datepicker): fix typos in datepicker demo

### PR DESCRIPTION
**Pull request by zladuric**
Fix a typo in the locale example code (import `deLocale` instead of `de`).
Fix a typo in datepicker description.

# PR Checklist
Before creating new PR, please take a look at checklist below to make sure that you've done everything that needs to be done before we can merge it.

 - [x] read and followed the [CONTRIBUTING.md](https://github.com/valor-software/ngx-bootstrap/blob/development/CONTRIBUTING.md) guide.
 - [ ] built and tested the changes locally.
 - [ ] added/updated tests.
 - [ ] added/updated API documentation.
 - [ ] added/updated demos.

### CODE REVIEW

**Review by IlyaSurmay — DISMISSED** [review]

File: demo/src/app/components/+datepicker/datepicker-section.list.ts [hunk 1]
```diff
@@ -38,7 +38,7 @@ export const demoComponentContent: ContentSection[] = [
         html: require('!!raw-loader?lang=markup!./demos/bs-popup/date-picker-popup.html'),
         description: `
           <p><code>BsDatepickerModule</code> is activily developed but you can use it already</p>
-          <p>Notebale change is additional css for it <code> "/datepicker/bs-datepicker.css"</code> <br></p>
+          <p>Noteable change is additional css for it <code> "/datepicker/bs-datepicker.css"</code> <br></p>
```
Comment (valorkin) [inline demo/src/app/components/+datepicker/datepicker-section.list.ts]: Hey, I am pretty sure its 'notable' ;)

**Review by valorkin — CHANGES_REQUESTED** [review]
Thanks for help ;)

**Comment by zladuric** [comment]
Yup @valorkin, it's "notable", not any of the other two versions. Totally nailed this PR by fixing a typo with another typo :) Anyway, added another commit, or if you want it to be cleaner, I can just drop the whole PR and make a new one.

**Review by valorkin — APPROVED** [review]

**Comment by valorkin** [comment]
thanks :) merged,
will be on http://ngx-bootstrap.surge.sh/#/ soon

**Pull request merged**